packages/waku/src/lib/builder/build.ts

```diff
@@ -394,6 +395,14 @@ const buildClientBundle = async (
   return clientBuildOutput;
 };
 
+export const extractNonJsAssets = (buildOutput: RollupOutput) =>
```

Comment:
Please don't export.
```suggestion
const extractNonJsAssets = (buildOutput: RollupOutput) =>
```